<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Al Kotobiya</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/main.ico</normaloff>:/images/main.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="Morocco"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Arabic" country="Morocco"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="Arabic" country="Morocco"/>
    </property>
    <property name="title">
     <string>ملف</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="locale">
     <locale language="Arabic" country="Morocco"/>
    </property>
    <property name="title">
     <string>مساعدة</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>اعدادات</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>عرض</string>
    </property>
    <addaction name="actionIndexDock"/>
    <addaction name="actionSearchDock"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockIndexs">
   <property name="accessibleName">
    <string>الفهرس</string>
   </property>
   <property name="accessibleDescription">
    <string>الفهرس</string>
   </property>
   <property name="locale">
    <locale language="Arabic" country="Morocco"/>
   </property>
   <property name="windowTitle">
    <string>الفهرس</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>184</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>السور :</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="listView">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>34</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>الآية :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxAyaNumber">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>43</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>الصفحة :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxPageNumber">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> / 604</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>604</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSearch">
   <property name="toolTip">
    <string>نافذة البحث في القرآن الكريم</string>
   </property>
   <property name="windowTitle">
    <string>البحث</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarGeneral">
   <property name="windowTitle">
    <string>عام</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewTab"/>
   <addaction name="separator"/>
   <addaction name="actionIndexDock"/>
   <addaction name="actionSearchDock"/>
  </widget>
  <widget class="QToolBar" name="toolBarNavigation">
   <property name="windowTitle">
    <string>التنقل</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevPage"/>
   <addaction name="actionPrevAYA"/>
   <addaction name="actionNextAYA"/>
   <addaction name="actionNextPage"/>
  </widget>
  <widget class="QToolBar" name="toolBarTafesir">
   <property name="windowTitle">
    <string>التفاسير</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>الخروج</string>
   </property>
   <property name="toolTip">
    <string>اغلاق البرنامج</string>
   </property>
   <property name="statusTip">
    <string>الخروج من البرنامج</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>حول البرنامج</string>
   </property>
   <property name="statusTip">
    <string>معلومات حول البرنامج</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>تعديلات...</string>
   </property>
   <property name="iconText">
    <string>تعديلات...</string>
   </property>
   <property name="toolTip">
    <string>تعديل خصائص البرنامج</string>
   </property>
   <property name="statusTip">
    <string>تعديل خصائص البرنامج</string>
   </property>
  </action>
  <action name="actionIndexDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/edit_fahrass.png</normaloff>:/menu/images/edit_fahrass.png</iconset>
   </property>
   <property name="text">
    <string>نافذة الفهرس</string>
   </property>
   <property name="statusTip">
    <string>اظهار أو اخفاء نافذة الفهرس</string>
   </property>
  </action>
  <action name="actionSearchDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/find.png</normaloff>:/menu/images/find.png</iconset>
   </property>
   <property name="text">
    <string>نافذة البحث</string>
   </property>
   <property name="statusTip">
    <string>اظهار أو اخفاء نافذة البحث في القرآن الكريم</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/bookmark-new.png</normaloff>:/menu/images/bookmark-new.png</iconset>
   </property>
   <property name="text">
    <string>تبويب جديد</string>
   </property>
   <property name="statusTip">
    <string>فتح تبويب جديد</string>
   </property>
  </action>
  <action name="actionNextAYA">
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/go-first.png</normaloff>:/menu/images/go-first.png</iconset>
   </property>
   <property name="text">
    <string>الاية التالية</string>
   </property>
   <property name="toolTip">
    <string>الاية التالية</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/go-previous.png</normaloff>:/menu/images/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>الصفحة التالية</string>
   </property>
   <property name="toolTip">
    <string>الصفحة التالية</string>
   </property>
  </action>
  <action name="actionPrevAYA">
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/go-last.png</normaloff>:/menu/images/go-last.png</iconset>
   </property>
   <property name="text">
    <string>الاية السابقة</string>
   </property>
   <property name="toolTip">
    <string>الاية السابقة</string>
   </property>
  </action>
  <action name="actionPrevPage">
   <property name="icon">
    <iconset>
     <normaloff>:/menu/images/go-next.png</normaloff>:/menu/images/go-next.png</iconset>
   </property>
   <property name="text">
    <string>الصفحة السابقة</string>
   </property>
   <property name="toolTip">
    <string>الصفحة السابقة</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
